### PR TITLE
Add migration doc

### PR DESCRIPTION
**Pull request by bradmanAWS**
**Issue #:** 

**Description of changes:**
Add a migration doc for V2

**Testing**
1. Have you successfully run `npm run build:release` locally?

2. How did you test these changes?

3. If you made changes to the component library, have you provided corresponding documentation changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by michhyun1** [comment]
Just curious why are we merging it into a branch called express?

**Comment by bradmanAWS** [comment]
> Just curious why are we merging it into a branch called express?

The express branch contains all the breaking changes for v2. They were kept out of master to not block if any hotfixes were needed.

**Comment by richnew10** [comment]
@bradmanAWS shouldn't this be linked from the README?

**Comment by bradmanAWS** [comment]
> @bradmanAWS shouldn't this be linked from the README?

Ideally, but the repo is already using ghpages to host storybook, so I wasn't sure where this should live. I was going to let the SDK team decide where it should live.

File: Migrating_to_V2.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## What’s New
+
+- Updated interface for `ChatBubble` component.
+- Using `forwardRef` in `ChatBubbleContainer` component.
+- Using `forwardRef` in `Textarea` component.
+- New `useDevicePermissionStatus` hook exported.
+- A callback parameter has been added to `realtimeUnsubscribeFromVolumeIndicator` hook.
+- Add click handler and image preview to `MessageAttachment` component.
+- Add optional `badge` prop to icon `Button`.
```
Comment (p-foucht) [inline Migrating_to_V2.md:30]: Can you call out that the js-sdk dependency has been updated to only support 2.x or w/e it is?

Comment (bradmanAWS) [reply Migrating_to_V2.md:30]: Great call out

**Review by p-foucht — APPROVED** [review]

**Comment by bradmanAWS** [comment]
Closing for https://github.com/aws/amazon-chime-sdk-component-library-react/pull/400 to avoid merge conflicts

**Pull request closed without merging**